yarn-project/noir-contracts/src/contracts/pokeable_token_contract/src/main.nr

```diff
@@ -0,0 +1,138 @@
+mod storage;
+mod address_note;
+
+contract PokeableToken {
```

Comment:
Curious if you tried using LSP when writing this

Reply:
I have not. Do we have some docs how to set it up?

Reply:
Not really yet officially as its been hacked quite a bit to work, but all you need is recent noir installed with noirup and the newer extension (noir language support)
<img width="380" alt="Screenshot 2023-07-20 at 1 29 42 PM" src="https://github.com/AztecProtocol/aztec-packages/assets/163993/77c8cfc9-6c7c-45aa-972d-d27a879c8544">

Reply:
Cool, will set it up tomorrow. Thanks